process/block/baseProcess.go

```diff
@@ -2200,3 +2201,28 @@ func (bp *baseProcessor) checkSentSignaturesAtCommitTime(header data.HeaderHandl
 
 	return nil
 }
+
+func isProofEmpty(proof data.HeaderProofHandler) bool {
```

Comment:
we can add this check when adding a new proof, we should not add an empty proof to pool

Reply:
I think the check is already being done on the interceptor in the integrity check
```func (iep *interceptedEquivalentProof) integrity() error```